bin/rule.ml

```diff
@@ -57,22 +61,18 @@ let print_rule ~nd ~prelude ~md_file ~ml_files ~dirs ~root options =
     let files = String.Set.of_list (List.map prelude_file prelude) in
     String.Set.elements files
     |> List.map (fun f -> Fmt.strf "         %s" f)
-    |> String.concat ~sep:"\n"
   in
   let root = match root with None -> "" | Some r -> Fmt.strf "--root=%s " r in
   let deps =
-    let sep1 = if var_names <> [] then "\n" else "" in
-    let sep2 = if dirs <> [] && prelude <> "" then "\n" else "" in
-    let x = Fmt.strf "%a%s%a%s%s"
-        Fmt.(list ~sep:(unit "\n") pp_ml_deps) (List.combine var_names ml_files)
-        sep1
-        Fmt.(list ~sep:(unit "\n") pp_dir_deps) dirs
-        sep2
-        prelude
+    let x =
```

Comment:
If we start refactoring this part how about a better name than `x` like `depends` maybe?

Reply:
It's not necessary at all. This part of the code need to be cleaned up.

Reply:
The fact that it's going to be refactored later doesn't mean we shouldn't properly name variables now. It's rather the opposite actually, making the code easier to understand helps with later refactoring.

I don't mind if you leave it like that as it is in a way unrelated to this PR. I just think it's good practice to improve the code we're modifying.

Reply:
I removed the binding. My second sentence was not an argument but a complain.
I'm working on improving that module.